### ISSUE TITLE
build error

### ISSUE THREAD

**Issue by junhyeokahn** [DAIRLab/dairlib-public#1]
Hello,
Thanks for maintaining this repository.

While I am building it (```bazel build``` in the dairlib-public ), I got the following error.

```
ERROR: /Users/junhyeokahn/Repository/dairlib-public/systems/trajectory_optimization/BUILD.bazel:9:1: C++ compilation of rule '//systems/trajectory_optimization:dircon' failed (Exit 1)
In file included from systems/trajectory_optimization/dircon.cc:1:
In file included from systems/trajectory_optimization/dircon.h:4:
In file included from systems/trajectory_optimization/dircon_opt_constraints.h:4:
systems/trajectory_optimization/dircon_kinematic_data.h:5:10: fatal error: 'gflags/gflags.h' file not found
#include <gflags/gflags.h>
         ^~~~~~~~~~~~~~~~~
1 error generated.
INFO: Elapsed time: 9.963s, Critical Path: 9.32s
INFO: 11 processes: 9 darwin-sandbox, 2 local.
FAILED: Build did NOT complete successfully
```
Did I miss some installing procedure?

**Comment by mposa** [comment]
Did you run the appropriate `./install_prereqs` for Drake during the Drake installation process?

By the way, I want to warn you that this branch is still in a development stage. Right now, it requires that Drake can build with SNOPT and will fail (at runtime) if it cannot. To remove this requirement just delete any references to SNOPT in the examples, and solve via Drake's default solver methods. This should then default to IPOPT.

**Comment by junhyeokahn** [comment]
Yes, I am using Mac OS and run corresponding ```./install_prereqs```

I am getting the error in compile stage regarding to the gflags path.

**Comment by mposa** [comment]
I would check to make sure that the gflags package is properly installed by homebrew. There are also many examples in Drake itself that use gflags. If they also fail to build, I'd recommend pinging the Drake developers (on https://stackoverflow.com/questions/tagged/drake for instance)

**Comment by junhyeokahn** [comment]
gflags are installed correctly by homebrew and I had no problem in running examples in Drake.

**Comment by mposa** [comment]
I think I see the issue, though it's odd that it does not seem to cause issues compiling on Ubuntu. I just pushed a commit https://github.com/DAIRLab/dairlib-public/commit/4e8464a53df6fcf87ce5c8f094c262e9e9af7335 to hopefully address it.
